### ISSUE TITLE
Additional prefix for firewalld log denied

### ISSUE THREAD

**Issue by unknowndevQwQ** [firewalld/firewalld#1096]
### What would you like to be added

Add a `firewalld-` suffix to the discard after rejection log, e.g. `ct state invalid log prefix "firewalld-STATE_INVALID_DROP: "`.

### Why is this needed

It is easier to find firewalld log-deine logs in a large number of logs.

Off-topic: Please turn on the github discussion function for normal discussion use.

**Comment by erig0** [comment]
> Add a `firewalld-` suffix to the discard after rejection log, e.g. `ct state invalid log prefix "firewalld-STATE_INVALID_DROP: "`.

The prefix should probably be configurable.

> Off-topic: Please turn on the github discussion function for normal discussion use.

Done.

**Comment by unknowndevQwQ** [comment]
> > Add a `firewalld-` suffix to the discard after rejection log, e.g. `ct state invalid log prefix "firewalld-STATE_INVALID_DROP: "`.
> 
> The prefix should probably be configurable.

At the moment I don't find a corresponding way to set it in firewall-config.

**Comment by erig0** [comment]
> > > Add a `firewalld-` suffix to the discard after rejection log, e.g. `ct state invalid log prefix "firewalld-STATE_INVALID_DROP: "`.
> > 
> > 
> > The prefix should probably be configurable.
> 
> At the moment I don't find a corresponding way to set it in firewall-config.

Right. I meant for this feature request we should make it configurable.
